### DISCUSSION TITLE
Right to left support

### DISCUSSION THREAD

**Discussion by motahharm** [tailwindlabs/tailwindcss#7261] [General]
Hello to you and your good team

Want to add right-to-left support to Tailwind?

Can I help?

**Comment by adamwathan** [comment]
Hey! We have RTL and LTR modifiers for making direction-specific adjustments already, can read about them here:

https://tailwindcss.com/docs/hover-focus-and-other-states#rtl-support

**Comment by motahharm** [comment]
hey @adamwathan 
Thank You So match
